crates/re_renderer/src/renderer/rectangles.rs

```diff
@@ -83,6 +89,9 @@ pub struct ColormappedTexture {
     /// Setting a color mapper for a four-component texture is an error.
     /// Failure to set a color mapper for a one-component texture is an error.
     pub color_mapper: Option<ColorMapper>,
+
+    /// For textures that don't store color information in conventional RGB color space, you need to supply a TextureEncoding.
+    pub encoding: Option<TextureEncoding>,
```

Comment:
`encoding: None` feels a bit odd to me since everything is encoded somehow. But usually we specify this via the texture format. I'd be in favor of calling it `shader_decoding` (with the type accordingly renamed) to point out that we're instructing the shader to do decoding on the fly.

Related to comment further down on gamma being incorrect.

Reply:
I do agree on the naming here, shader_decoding makes more sense, thanks for te suggestion!